### PR TITLE
Shows restricted elements if present

### PR DESCRIPTION
**Pull request by Ananthavijay**
Fixes https://github.com/CircuitVerse/CircuitVerse/issues/398

#### Describe the changes you have made in this pr -

- Added an if condition.

### CODE REVIEW

File: app/views/assignments/show.html.erb [hunk 1]
```diff
@@ -30,10 +30,12 @@
...
 
+     <% if @assignment.restricted_circuit_elements != "None" %>
       <p>
         <strong>Description:</strong>
         <p><%= @assignment.description&.html_safe%></p><br>
       </p>
+    <% end %>
 
       <p>
         <strong>Restricted Circuit Elements:</strong>
```
Comment (fluctix) [inline app/views/assignments/show.html.erb]: Shouldnt this be within the if condition?

Comment (fluctix) [reply app/views/assignments/show.html.erb]: It seems you have added the condition to the description...

Comment (MrBartusek) [reply app/views/assignments/show.html.erb]: I haven't seen that too, good point

**Comment by fluctix** [comment]
That's the reason appveyor has gone red

**Comment by Ananthavijay** [comment]
My bad, didn't see it properly

**Comment by MrBartusek** [comment]
@fluctix yes it is `Failure/Error: expect(response.body).to include(@assignment.description)`

**Comment by Ananthavijay** [comment]
Pushed it in a hurry, thanks for spotting it out @fluctix

**Comment by fluctix** [comment]
Looks good..

**Pull request merged**